### PR TITLE
Add the raw data, data transformation script, and final dataset.

### LINKED ISSUE #2: Transform raw data into the dataset we require
This is a one time job, will write code in RMD.

### PR DESCRIPTION
**Pull request by cath-parkinson**
Closes #2

### CODE REVIEW

File: data-raw/Transform.Rmd [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Takes the ONS dataset and produces the data needed for the app.
+
+Data source: https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/livebirths/datasets/babynamesinenglandandwalesfrom1996
+
+This data only runs up to 2021. We will update once then latset data is available.
```
Comment (HorridTom) [inline data-raw/Transform.Rmd:15]: Some spelling errors on this line

File: data-raw/Transform.Rmd [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                  year = year) %>% 
+    dplyr::mutate(count = ifelse(grepl("\\[x\\]", count), "0", count)) %>% 
+    dplyr::mutate(gender = gender_string) %>% 
+    dplyr::mutate(count = as.integer(count),
+                  year = as.integer(year),
+                  gender = factor(gender,
+                                  levels = c("Girl",
+                                             "Boy"))) %>% 
+    dplyr::select(name, gender, year, count)
+
```
Comment (HorridTom) [inline data-raw/Transform.Rmd:61]: Are we using explicit returns in general for this repo?

**Review by HorridTom — CHANGES_REQUESTED** [review]
Mostly fine.

**Comment by cath-parkinson** [comment]
Will fix typos and conflicts in manual pull.

**Pull request merged**